lib/krikri/provider_list.rb

```diff
@@ -0,0 +1,19 @@
+module Krikri
+  # Contrtructs a list of providers
+  class ProviderList
+
+    def initialize
+      @index_querier = Krikri::IndexQuerier.new
```

Comment:
Maybe give the option to inject this?

```ruby
def initialize(index_querier = Krikri::IndexQuerier.new)
  @index_querier = index_querier
end
```

(I think I agree w/ @anarchivist that that class could use a better name).